### PR TITLE
changed 8080 to 80 to fix issue #194

### PR DESCRIPTION
**Pull request by saurabh30**
I have just replaced 8080 by 80 everywhere.

### CODE REVIEW

**Comment by karuppiah7890** [comment]
Thanks @saurabh30 for the PR! :smile: 
And actually, if the HTTP server's port is 80, we don't need to specify it in the URL, so you can just put `http://localhost` instead of `http://localhost:80`

File: docs/local_deploy.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ After this, you'll need to create your own .env file. Check out the .env.example
 
 Now run `docker-compose up`. This will run all the containers needed to launch this app, and will take some time to finish.
 
-When the process is finished, the app will be exposed on port 8080 and accessible by visiting `http://[hostname]:8080`. You may use a reverse proxy such as Nginx to serve the app on port 80.
+When the process is finished, the app will be exposed on port 80 and accessible by visiting `http://[hostname]:80`. You may use a reverse proxy such as Nginx to serve the app on port 80.
```
Comment (karuppiah7890) [inline docs/local_deploy.md]: @saurabh30 You can omit the line which says "You may use a reverse proxy..." as the app is already being served at port 80 :)

Comment (karuppiah7890) [inline docs/local_deploy.md]: `http://[hostname]` should be enough

File: docs/local_deploy.md [hunk 2]
```diff
@@ -22,12 +22,12 @@ After this, you'll need to create your own .env file. Check out the .env.example
...
 
 ###### Installation summary
 
 1. Install and run the Docker daemon.
 2. Clone the repository and change into it `git clone https://github.com/freeCodeCamp/Mail-for-Good && cd Mail-for-Good`.
 3. Run `cp .env.example .env` then open `.env` and edit it, passing in your own values.
 4. Run `docker-compose up`. Wait for it to finish.
-5. Visit `http://[hostname]:8080` or `localhost:8080` if running locally.
+5. Visit `http://[hostname]:80` or `localhost` if running locally.
```
Comment (karuppiah7890) [inline docs/local_deploy.md]: Here too `http://[hostname]` should be enough :)

**Comment by zhakkarn** [comment]
It also seems good to me. 
@saurabh30, do you mind if I squash these commits when merging them?

**Comment by karuppiah7890** [comment]
Same here, LGTM! :) Thanks @saurabh30

**Comment by AndrewGHC** [comment]
Lgtm, cheers

**Pull request merged**